components/match2/commons/starcraft_starcraft2/match_group_input_starcraft_ffa.lua

```diff
@@ -44,49 +44,45 @@ function StarcraftFfaInput.adjustData(match)
 	match.noscore = noscore
 
 	--process pbg entries and set them into match.pbg (will get merged into extradata later on)
-	match = StarcraftFfaInput.getPbg(match)
+	match = StarcraftFfaInput._getPbg(match)
 
 	--parse opponents + determine match mode + set initial stuff
 	match.mode = ''
-	match, OppNumber = StarcraftFfaInput.OpponentInput(match, OppNumber, noscore)
+	match, OppNumber = StarcraftFfaInput._opponentInput(match, OppNumber, noscore)
```

Comment:
Why is OppNumber a parameter here? It will always be 0?
Also it's not in camelCase, and it could be more descriptive. eg `numberOfOpponents`